codepropertygraph/src/main/scala/io/shiftleft/passes/CpgPass.scala

```diff
@@ -39,6 +42,7 @@ import io.shiftleft.codepropertygraph.Cpg
   */
 abstract class CpgPass(cpg: Cpg) {
   import CpgPass.logger
+  implicit val executionContext: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
(minor) i think `import scala.concurrent.ExecutionContext.Implicits.global` does the same, no need for a local implicit val

Reply:
Yes, I've removed it